ElmersAPIServiceTests/CoffeeTests.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using ElmersAPIService;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ElmersAPIService.Tests
+{
+    [TestClass()]
+    public class CoffeeTests
+    {
+        [TestMethod()]
+        public void ProcessCoffeeValidInput()
```

Comment:
We could add DataRowAttributes here and pass in the data to test and the expected result as input

Reply:
Added DynamicData instead of DataRow attribute to handle the custom CoffeeSettings object that is the expected result